Sources/Hummingbird/Middleware/MetricsMiddleware.swift

```diff
@@ -49,17 +49,17 @@ public struct MetricsMiddleware<Context: RequestContext>: RouterMiddleware {
             // Don't record uri in 404 errors, to avoid spamming of metrics
             if let endpointPath = context.endpointPath {
                 dimensions = [
-                    ("hb_uri", endpointPath),
-                    ("hb_method", request.method.rawValue),
+                    ("uri", endpointPath),
+                    ("method", request.method.rawValue),
                 ]
-                Counter(label: "hb_requests", dimensions: dimensions).increment()
+                Counter(label: "hb.requests", dimensions: dimensions).increment()
```

Comment:
```suggestion
                Counter(label: "http.server.active_requests", dimensions: dimensions).increment()
```

Reply:
This is the number of active requests, not the total number